sampletester/inputs.py

```diff
@@ -69,6 +69,14 @@ def index_docs(*file_patterns: str) -> parser.IndexedDocs:
   if not file_patterns:
     file_patterns = ['**/*.yaml']
 
+  def log_files(indexed_files):
```

Comment:
nit: this method doesn't seem to modify `indexed_files`. It might make sense to have this method return `None`. Not a requirement, the code is correct too.

Reply:
I have it like this so the caller can simply `return log_files(indexed_files)` instead of
```
log_files(indexed_files)
return indexed_files
```

Reply:
Make sense.